src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -879,6 +879,15 @@ public virtual bool CanApplyChange(ApplyChangesKind feature)
         }
 
         /// <summary>
+        /// Returns <code>true</code> if a reference to referencedProject can be added to 
+        /// referencingProject.  <code>false</code> otherwise.
+        /// </summary>
+        internal virtual bool CanAddProjectReference(ProjectId referencingProject, ProjectId referencedProject)
+        {
+            return false;
```

Comment:
Looking at the rest of the workspace, it looks like this is what we generally do in the root workspace.